.gitignore

```diff
@@ -0,0 +1,3 @@
+.DS_Store
```

Comment:
You can set up a global . gitignore file that lists files that will be ignored on all Git projects. So you need to delete this OS/IDE specific files from here and set them globally.

---
https://www.mashupgarage.com/playbook/git/gitignore.html#:~:text=You%20can%20set%20up%20a,files%2C%20and%20other%20miscellaneous%20crap.